lang/po/Firmware_nl.po

```diff
@@ -1837,66 +2194,104 @@ msgstr ""
+#: ../../Firmware/messages.cpp:158 ../../Firmware/ultralcd.cpp:5863
 msgid "Timeout"
 msgstr "Time-out"
 
 #. MSG_TOTAL c=6
-#: ../../Firmware/messages.cpp:97 ../../Firmware/ultralcd.cpp:1157
-#: ../../Firmware/ultralcd.cpp:1305
+#: ../../Firmware/messages.cpp:101 ../../Firmware/ultralcd.cpp:1167
+#: ../../Firmware/ultralcd.cpp:1300
 msgid "Total"
 msgstr "Totaal"
 
 #. MSG_TOTAL_FAILURES c=20
-#: ../../Firmware/messages.cpp:98 ../../Firmware/ultralcd.cpp:1200
-#: ../../Firmware/ultralcd.cpp:1226 ../../Firmware/ultralcd.cpp:1336
+#: ../../Firmware/messages.cpp:102 ../../Firmware/ultralcd.cpp:1214
+#: ../../Firmware/ultralcd.cpp:1244 ../../Firmware/ultralcd.cpp:1330
 msgid "Total failures"
 msgstr "Totaal fouten"
 
 #. MSG_TOTAL_FILAMENT c=19
-#: ../../Firmware/ultralcd.cpp:2395
+#: ../../Firmware/ultralcd.cpp:2385
 msgid "Total filament"
 msgstr "Totaal fil."
 
 #. MSG_TOTAL_PRINT_TIME c=19
-#: ../../Firmware/ultralcd.cpp:2396
+#: ../../Firmware/ultralcd.cpp:2386
 msgid "Total print time"
 msgstr "Totaal printtijd"
 
 #. MSG_TUNE c=18
-#: ../../Firmware/ultralcd.cpp:5508
+#: ../../Firmware/ultralcd.cpp:5509
 msgid "Tune"
 msgstr "Fijnafstemming"
 
-#. MSG_UNLOAD_FILAMENT c=18
-#: ../../Firmware/messages.cpp:111 ../../Firmware/ultralcd.cpp:5572
-#: ../../Firmware/ultralcd.cpp:5586
+#. MSG_TITLE_UNLOAD_MANUALLY c=20
+#: ../../Firmware/mmu2/errors_list.h:153 ../../Firmware/mmu2/errors_list.h:191
+msgid "UNLOAD MANUALLY"
+msgstr "ONTLAAD MANUEEL"
+
+#. MSG_DESC_UNLOAD_MANUALLY c=20 r=8
+#: ../../Firmware/mmu2/errors_list.h:233 ../../Firmware/mmu2/errors_list.h:271
+msgid ""
+"Unexpected FINDA reading. Ensure no filament is under FINDA and the selector "
+"is free. Check FINDA connection."
+msgstr ""
+"Onverwachte FINDA-aflezing. Zorg ervoor dat er geen filament onder FINDA zit"
+" en dat de selecteur vrij is. Controleer de FINDA-verbinding."
+
+#. MSG_BTN_UNLOAD c=6
+#: ../../Firmware/mmu2/errors_list.h:284 ../../Firmware/mmu2/errors_list.h:294
+msgid "Unload"
+msgstr "Ontla."
+
+#. MSG_UNLOAD_FILAMENT c=16
+#: ../../Firmware/messages.cpp:115 ../../Firmware/ultralcd.cpp:5574
+#: ../../Firmware/ultralcd.cpp:5591
 msgid "Unload filament"
 msgstr "Fil. uitwerpen"
 
 #. MSG_UNLOADING_FILAMENT c=20
-#: ../../Firmware/messages.cpp:112 ../../Firmware/mmu.cpp:957
-#: ../../Firmware/ultralcd.cpp:5205
+#: ../../Firmware/Marlin_main.cpp:3502 ../../Firmware/messages.cpp:116
+#: ../../Firmware/mmu2_progress_converter.cpp:50
+#: ../../Firmware/ultralcd.cpp:5193
 msgid "Unloading filament"
 msgstr "Uitwerpen filament"
 
+#. MSG_PROGRESS_UNLOAD_FINDA c=20
+#: ../../Firmware/mmu2_progress_converter.cpp:11
+#: ../../Firmware/mmu2_progress_converter.cpp:37
+msgid "Unloading to FINDA"
+msgstr "Ontlade naar FINDA"
+
+#. MSG_PROGRESS_UNLOAD_PULLEY c=20
+#: ../../Firmware/mmu2_progress_converter.cpp:12
+#: ../../Firmware/mmu2_progress_converter.cpp:38
+msgid "Unloading to pulley"
+msgstr "Ontlade n. riemschi."
```

Comment:
consistency

Reply:
What should we use for Pulley and Idler in Dutch?

Reply:
I don't think it matters too much so long as we are consistent that pulley is translated one way and idler another.